auto_nag/scripts/workflow/no_severity.py

```diff
@@ -194,7 +194,19 @@ def get_bz_params(self, date):
                     "f18": "component",
                     "o18": "everchanged",
                     "f19": "CP",
-                    "f20": "CP",
+                    "j20": "OR",
+                    "f20": "OP",
+                    "f21": "bug_severity",
+                    "o21": "changedfrom",
+                    "v21": "critical",
+                    "f22": "bug_severity",
+                    "o22": "changedfrom",
+                    "v22": "major",
+                    "f23": "bug_severity",
+                    "o23": "changedfrom",
+                    "v23": "blocker",
+                    "f24": "CP",
+                    "f30": "CP",
```

Comment:
Why the jump from 24 to 30?

Reply:
Just make space if we want to add other stuff later. This will allow adding without modifying other parts as we are doing now.